torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -1520,13 +1525,13 @@ def make_allocation(
         size = self.codegen_shape_tuple(shape)
         stride = self.codegen_shape_tuple(orig_stride)
         if config.abi_compatible:
-            size_array_var = self.codegen_int_array_var(
+            size_array_var = self.codegen_var_array(
```

Comment:
Because codegen_int_array_var was used for generating size_array, I prefer to keep its own int_array_id for naming. You can add `codegen_var_array` as a higher level and call `codegen_int_array_var` for int array.

Reply:
I see. Let me update it.

Reply:
I expect to keep calling `codegen_int_array_var` here and other similar places when we are generating int array for sizes or strides.